src/test/kotlin/StringAddCalculatorTest.kt

```diff
@@ -0,0 +1,58 @@
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class StringAddCalculatorTest {
+    private lateinit var calculator: StringAddCalculator
+
+    @BeforeEach
+    fun setUp() {
+        calculator = StringAddCalculator();
+    }
+
+    @DisplayName(value = "빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.")
+    @ParameterizedTest
+    @NullAndEmptySource
+    fun emptyOrNull(text: String?) {
+        assertThat(calculator.add(text)).isZero();
```

Comment:
```suggestion
        assertThat(calculator.add(text)).isZero()
```
힌트 코드이기는하나 `ktlintCheck`를 확인하는 습관을 들이는 것이 좋습니다.

Reply:
힌트코드는 너무 당연하게 좋은 코드라고 생각했었네요
감사합니다!